src/api/entrypoints/alunos/views.py

```diff
@@ -83,3 +82,17 @@ async def get_aluno_email(aluno_email: str, repository=Depends(get_repo())):
 @router.get("/orientador/{orientador_id}", response_model=List[AlunoInDB])
 async def get_alunos_por_orientador(orientador_id: int, repository=Depends(get_repo())):
     return await ServicoAluno(repository).buscar_alunos_por_orientador(orientador_id)
+
+@router.put("/remover-orientador/{aluno_id}", response_model=AlunoInDB)
```

Comment:
O id está sendo passado na rota nesse caso. Na task é solicitado que seja no body.
```suggestion
@router.put("/remover-orientador/", response_model=AlunoInDB)
```

Reply:
Funcionou normalmente, mas alterei para o body, como requisitado na task.